.idea/libraries/org_antlr_antlr4_4_5_1_1.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
I don't have direct experience using Maven anyway, I usually use it through IDEs.